internal/app/mapper/mapper.go

```diff
@@ -89,3 +123,21 @@ func getResult(fragmentRule *rule) string {
 	stringFragment := fragmentRule.GetResult().GetStringFragment()
 	return stringFragment
 }
+
+func compare(requestNodeMatch *aggregationv1.MatchPredicate_RequestNodeMatch, nodeValue string) (bool, error) {
+	exactMatch := requestNodeMatch.GetExactMatch()
+	if exactMatch != "" {
```

Comment:
We would get a false negative if the nodeValue is empty right? Perhaps that is desired, but something to consider. Regardless would add a test case for this scenario.

Reply:
i think that is desired. Added some tests now.